Tests/js/views/assets/ez-view-tests.js

```diff
@@ -36,7 +36,56 @@ YUI.add('ez-view-tests', function (Y) {
                 view.get('subPlainView').get('active'),
                 "The active attribute of the subPlainView should be undefined"
             );
-        }
+        },
+    });
+
+    addEventsTest = new Y.Test.Case({
+        name: "eZ View addEvent tests",
+
+        setUp: function () {
+            this.TestView = Y.Base.create('TestView', Y.eZ.View, [], {
+                _something: function() {},
+            }, {});
+            this.event1 = {'.something': {
+                'tap': '_something'
+            }};
+            this.event2 = {'.whatever': {
+                'tap': '_something'
+            }};
+            this.view = new this.TestView({
+                events: this.event1,
+            });
+        },
+
+        _testAddEventsInDom: function () {
+            this.view._addDOMEventHandlers(this.event2);
```

Comment:
it's a protected method, you can not call it directly. Instead, you should define one or several view contructor(s) (like `this.TestView`) which use `_addDOMEventHandlers` in different cases (after the container has been initialized or in the case where the container is not initialized)